### PR TITLE
doc: emphasize name should be unique

### PR DESCRIPTION
**Pull request by xiang90**
Fix https://github.com/coreos/etcd/issues/3025

### CODE REVIEW

File: Documentation/configuration.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 + Human-readable name for this member.
 + default: "default"
 + env variable: ETCD_NAME
-+ This value is referenced as this node's own entries listed in the `-initial-cluster` flag (Ex: `default=http://localhost:2380` or `default=http://localhost:2380,default=http://localhost:7001`). This needs to match the key used in the flag if you're using [static bootstrapping](clustering.md#static).
++ This value is referenced as this node's own entries listed in the `-initial-cluster` flag (Ex: `default=http://localhost:2380` or `default=http://localhost:2380,default=http://localhost:7001`). This needs to match the key used in the flag if you're using [static bootstrapping](clustering.md#static). When using discovery, each member must have a unique name. `Hostname` or `machine-id` can be a good choice.
```
Comment (yichengq) [inline Documentation/configuration.md]: It always needs a unique name in all cases, right?

**Comment by robszumski** [comment]
LGTM

File: Documentation/configuration.md (hunk 1 above)
Comment (xiang90) [inline Documentation/configuration.md]: not really. restarting an member or rejoin into a cluster does not require a unique name.

Comment (yichengq) [inline Documentation/configuration.md]: ok. nit: Could we put this in a new paragraph?

Comment (xiang90) [inline Documentation/configuration.md]: That will break the markdown. I will keep this as it is since I feel this is clear.

**Pull request merged**